pypiserver/templates/deployment.yaml

```diff
@@ -35,12 +35,16 @@ spec:
         - name: {{ . }}
       {{- end }}
       {{- end }}
-      {{- if .Values.persistence.enabled }}
       securityContext:
+        runAsUser: {{ .Values.securityContext.runAsUser }}
+        runAsGroup: {{ .Values.securityContext.runAsGroup }}
+        {{- if .Values.persistence.enabled }}
```

Comment:
While your are adding values to the security context block, could you revamp it a little bit to follow the recommendation found here: https://docs.bitnami.com/tutorials/production-ready-charts/? Something like that:
```
{{- if .Values.securityContext.enabled }}
securityContext:
  runAsUser: {{ .Values.securityContext.runAsUser }}
  runAsGroup: {{ .Values.securityContext.runAsGroup }}
  fsGroup: {{ .Values.securityContext.fsGroup }}
{{- end }}
```